.github/workflows/nano_howto_guides_tests.yml

```diff
@@ -72,6 +72,28 @@ jobs:
           $CONDA/bin/conda remove -n howto-guides-inference-pytorch-openvino --all
         env:
           ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+      
+      - name: Run unit tests for Inference PyTorch (InferenceOptimizer) how-to guides
+        shell: bash
+        run: |
+          $CONDA/bin/conda create -n howto-guides-inference-pytorch-openvino -y python==3.7.10 setuptools=58.0.4
+          source $CONDA/bin/activate howto-guides-inference-pytorch-openvino
+          $CONDA/bin/conda info
+          bash python/nano/dev/build_and_install.sh linux default false pytorch
+          source bigdl-nano-init
+          pip install pytest nbmake
+          pip install ipykernel==5.5.6
+          pip install torch==1.12.0 torchvision --extra-index-url https://download.pytorch.org/whl/cpu 
+          pip install --upgrade intel-extension-for-pytorch
+          pip install onnx==1.12.0 onnxruntime==1.12.1 onnxruntime-extensions
+          pip install openvino-dev
+          pip install neural-compressor==1.12
+          pip install --upgrade numpy==1.21.6
```

Comment:
do we still need this one? since openvino-dev==2022.2.0 has a wide support to numpy

Reply:
oh right, numpy is redundant here. thanks :)